process/cla_faq.md

```diff
@@ -0,0 +1,21 @@
+# Contributor License Agreement (CLA) FAQ
+
+Many of the projects hosted by ASWF leverage a Contributor License Agreement for contributors to the given project. These projects use [EasyCLA](https://lfcla.com/) for automating the process of signing and managing CLAs for the project. 
+
+This document captures some commonly asked questions by organizations on CLAs and using the EasyCLA system.
+
+Q: Why does the project require me to sign a CLA?
+
+A: It depends from project to project, but generally the project is looking to ensure that each indivudual is properly authorized to contribute to the project ( most specifically by thier employer ) and that any additional terms that a contributor should be bound to.
+
+Q: What is the difference between a Corporate and Individual CLA?
+
+A: The Corporate CLA is designed for an organization to authorize contributors that have permission to contribute on behalf of the organization. The Individual CLA is specific for an indivudual contributor, including those not affliated with an organization.
+
```

Comment:
Q: Do I need to have my employers sign the CCLA if I am making contributions on my own time and using my own equipment?

A: It depends, but probably yes! In large tech and entertainment companies, full-time employees are nearly always required to sign an IP agreement at the start of their employment that assigns the company ownership of any IP you generate, 24/7. Unless you have a written agreement making an exception, either generally or for a specific project, the company probably owns all your output and they need to sign a CCLA. (Note: this advice is for employees. For contractors, the client usually only owns what you produce during the hours you bill for their project, but check your contract to be sure of the rules.)

Reply:
@lgritz Added this question in, with verbiage from @swinslow - thank you!